src/test/java/com/rabbitmq/client/test/TestUtils.java

```diff
@@ -31,4 +34,14 @@ public static ConnectionFactory connectionFactory() {
         return connectionFactory;
     }
 
+    public static void close(Connection connection) {
+        if(connection != null) {
+            try {
+                connection.close();
+            } catch (IOException e) {
+                throw new RuntimeException(e);
```

Comment:
The initial goal was to avoid a null check in tests, and save a couple of lines. I'd prefer to know if something goes wrong in test destroy methods, instead of silently drop errors.